docs/faq.rst

```diff
@@ -335,11 +335,18 @@ also supports the ``--glob-archives`` parameter.
 How do I remove files from an existing backup?
 ----------------------------------------------
 
-Say you now want to remove old logfiles because you changed your
-backup policy as described above. The only way to do this is to use
-the :ref:`borg_recreate` command to rewrite all archives with a
-different ``--exclude`` pattern. See the examples in the
-:ref:`borg_recreate` manpage for more information.
+A file is only removed from a BorgBackup repository if all archives that contain
+the file are deleted and the corresponding data chunks are removed from the
+repository There are two ways how to remove files from a repository:
+
+1. Use :ref:`borg_delete` to remove all archives that contain the files. This
+will of course delete everything in the archive, not only some files. Finally,
+run :ref:`borg_compact` to delete the archives' chunks from the repository.
+
+2. If you really want to remove only some specific files, you can run the
+:ref:`borg_recreate` command to rewrite all archives with a different
+``--exclude`` pattern. See the examples in the :ref:`borg_recreate` manpage for
+more information.
```

Comment:
Hmm, these methods are not 100% safe (regarding the removal of the information from the repo's segment files).

They will of course logically remove it, but there is some chance that compact won't remove the unused chunks of the removed file because the respective segment file is under the compaction threshold. This can happen especially if the removed file is rather small.

Reply:
to solve that, we need options to `borg compact` so we can force an arbitrary threshold (in this case: a 0 threshold).

Reply:
Do we have such an issue (compact with threshold) already? If not, I will create such an issue.

Of course, there is a chance that a chunk is not compacted/removed. I will add a note about this.

Reply:
not sure whether we have an issue or not, please check and create one if needed.

Reply:
Oh dear, I am very forgetful... We had #4674 which I solved introducing the threshold parameter... I will update the PR accordingly.